code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -42,116 +39,183 @@
+		ADD_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
+		xeno.anchored = TRUE
 
+		if (do_after(xeno, activation_delay, INTERRUPT_ALL | BEHAVIOR_IMMOBILE, BUSY_ICON_HOSTILE))
+			xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] rips open the guts of [carbon]!"), SPAN_XENOHIGHDANGER("We rip open the guts of [carbon]!"))
+			carbon.spawn_gibs()
+			playsound(get_turf(carbon), 'sound/effects/gibbed.ogg', 75, 1)
+			carbon.apply_effect(get_xeno_stun_duration(carbon, 0.5), WEAKEN)
+			carbon.apply_armoured_damage(get_xeno_damage_slash(carbon, base_damage + damage_scale * predalienbehavior.kills), ARMOR_MELEE, BRUTE, "chest", 20)
 
-	for(var/mob/M in view(xeno))
-		if(M && M.client)
-			shake_camera(M, 0.2 SECONDS, 1)
+			xeno.animation_attack_on(carbon)
+			xeno.spin_circle()
+			xeno.flick_attack_overlay(carbon, "tail")
 
-	apply_cooldown()
-	return ..()
+		playsound(owner, 'sound/voice/predalien_growl.ogg', 75, 0, status = 0)
 
-/datum/action/xeno_action/activable/devastate/use_ability(atom/target)
-	var/mob/living/carbon/xenomorph/xeno = owner
+		REMOVE_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
+		xeno.anchored = FALSE
+		unroot_human(carbon, TRAIT_SOURCE_ABILITY("Devastate"))
 
-	if (!action_cooldown_check())
-		return
+		xeno.visible_message(SPAN_XENODANGER("[xeno] rapidly slices into [carbon]!"))
 
-	if (!xeno.check_state())
-		return
+		return ..()
 
-	if (!isxeno_human(target) || xeno.can_not_harm(target))
-		to_chat(xeno, SPAN_XENOWARNING("We must target a hostile!"))
-		return
 
-	if (get_dist_sqrd(target, xeno) > 2)
-		to_chat(xeno, SPAN_XENOWARNING("[target] is too far away!"))
-		return
+/datum/action/xeno_action/onclick/feralrush/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/predatoralien = owner
 
-	var/mob/living/carbon/carbon = target
+	if (!action_cooldown_check())
+		return
 
-	if (carbon.stat == DEAD)
-		to_chat(xeno, SPAN_XENOWARNING("[carbon] is dead, why would we want to touch them?"))
+	if (!istype(predatoralien) || !predatoralien.check_state())
 		return
 
-	var/datum/behavior_delegate/predalien_base/behavior = xeno.behavior_delegate
-	if(!istype(behavior))
+	if (buffs_active)
+		to_chat(predatoralien, SPAN_XENOHIGHDANGER("We cannot stack this!"))
 		return
 
 	if (!check_and_use_plasma_owner())
 		return
 
-	ADD_TRAIT(carbon, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
 
-	if (ishuman(carbon))
-		var/mob/living/carbon/human/human = carbon
-		human.update_xeno_hostile_hud()
+	RegisterSignal(predatoralien, COMSIG_XENO_POST_TOUGHEN, PROC_REF(disable_feralrush))
+	addtimer(CALLBACK(src, PROC_REF(remove_effects)), duration)
 
+	predatoralien.add_filter("predalien_toughen", 1, list("type" = "outline", "color" = "#421313", "size" = 1))
+	predatoralien.balloon_alert(predatoralien, "we feel our muscles tense, making us faster and more durable for a time.", text_color = "#522020ff")
```

Comment:
Not sure a balloon alert is necessary here, we already have a filter. Send it to chat instead if you want some fluff text.